bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveMultiInstanceCommandClass.java

```diff
@@ -353,34 +334,7 @@ private void handleMultiChannelCapabilityReportResponse(SerialMessage serialMess
 			}
 
 			// Add all the command classes supported by this endpoint
-			for (int i = 0; i < serialMessage.getMessagePayload().length - offset - 3; i++) {
```

Comment:
Yeah. Sorry about that. Surprised me as well. The PR was submitted with just the database change. The refactoring was performed later. Git aparently presents all changes on the master in a PR, not just those present at the time of creating the PR. My bad?!